src/scenes/end.js

```diff
@@ -1,23 +1,82 @@
-import elements from "../elements.js";
+import options from "../options.js";
 
 export default class EndScene extends Phaser.Scene {
     constructor() {
         super({ key: "EndScene" });
+        this.gameReplay = {};
+        this.buildYourOwn = {};
     }
 
     preload() {}
 
     create(data) {
-        elements.appEnd.style.visibility = "visible";
-        elements.appEnd.style.display = "flex";
-        elements.appEnd.style.marginTop = "10%";
-        elements.endScoreDisplay.innerHTML = `Score: ${data.score}`;
-
-        elements.buttonGameReplay.onclick = () => {
-            elements.appEnd.style.visibility = "invisible";
-            elements.appEnd.style.display = "none";
+        this.createDOMElements(data.score);
+
+        this.gameReplay.setInteractive();
+        this.buildYourOwn.setInteractive();
+
+        this.gameReplay.on("pointerup", () => {
             this.scene.stop("EndScene");
             this.scene.start("GameScene");
-        };
+        });
+
+        this.buildYourOwn.on("pointerup", () => {
+            window.location.href = "https://www.codecademy.com/learn/learn-phaser";
+        });
+    }
+
+    createDOMElements(score) {
+        // add score text
+        this.add.text(options.windowWidth / 2 - 110, 60, `Score: ${score}`, {
+            fontFamily: "sans-serif",
+            fontSize: "48px",
+            fontStyle: "bold",
+            color: "#000",
+            align: "center",
+        });
+
+        // add subheding
```

Comment:
```suggestion
        // add subheading
```

...although I don't think these comments are super useful. A bunch of them have roughly the same information as member names, and the rest have text explaining what they are. Probably better to remove them.